Add workaround for customers using flutter V2 (#135)
aws-amplify/amplify-adminui: markdown/tests/flutter/flutter_step4.md

platform :ios, '13.0'
```

> If you are using Flutter v2 and encountering build errors involving minimum deployment targets, you may need to update your app's `Podfile` in the `ios` directory.

Reviewer: Suggestion to shorten line 27 & 28: 
```suggestion
For Flutter v2, you need to update the `ios/Podfile` to account for minimum deployment targets
```
